src/searching.md

```diff
@@ -95,86 +94,86 @@ Things to note from the above:
 ## Limiting to a field
 
 You can also ask Anki to match only if a particular field contains some
-text. Unlike the searches above, searching on fields requires an 'exact
-match' by default.
+text. Unlike the previous search examples, searching in fields requires an exact
+match by default.
 
 `front:dog`\
-find notes with a Front field of exactly "dog". A field that says "a
+finds notes with a Front field of exactly "dog". A field that says "a
 dog" will not match.
 
 `"animal front:a dog"`\
-find notes where the "Animal Front" field is exactly "a dog". The double quotes are
-mandatory, see [below](#matching-special-characters).
+finds notes where the "Animal Front" field is exactly "a dog". The double quotes are
+mandatory, see [this section](#matching-special-characters) of the page.
```

Comment:
This line is fine as-is. If it needs clarification, as mentioned above, it should be consistent with uses of "section" and "page" in the manual.

Reply:
```suggestion
mandatory, see [later in this section](#matching-special-characters).
```